tests/performance/apdex-score/test/page-objects/contacts.page.js

```diff
@@ -5,6 +5,7 @@ class ContactsPage extends Page {
   async loadContactList(settingsProvider) {
     const page = settingsProvider.getPage('contact-list');
     await super.loadAndAssertPage(page);
+    await super.toggleAirplaneMode('on');
```

Comment:
Is the idea to turn on the airplane mode after login? 
If yes, it's better to make a separate function like this:

```js
// In load.page.js
asyn turnOnAirplaneMode(settingsProvider) {
  const commonElements = settingsProvider.getCommonElements();
  const UI_ELEMENT = commonElements?.relaunchAppAssert || '//*[@text="People"]';
  await this.waitForDisplayedAndRetry(UI_ELEMENT);
  await super.toggleAirplaneMode('on');
}
```
You do something similar for relaunching the app.
Then call that new function after the login in `test.apdex.js`

Reply:
Great thanks